views/alert.erb

```diff
@@ -1,7 +1,12 @@
-<li class="bad">
-	<h4><%= alert.message['service'] %></h4>
-	<p>
-		<%= alert.message['description']['subject'] %>
-		<%= alert.message['description']['description'] %>
+<li class="bad<% if !alert.message['acknowledged'] %> unacknowledged<% end %>">
+  <h4>
+    <%= alert.message['service'] %>
+    <% if !alert.message['acknowledged'] %>
```

Comment:
Rather than looking at properties of a hash in the template can we add a method to the Alert module?